requirements.txt

```diff
@@ -4,3 +4,4 @@ beautifulsoup4==4.11.1
 boto3==1.21.22
 ConfigParser==3.5.0
 validators==0.11.2
+ctap-keyring-device@git+https://github.com/kholia/ctap-keyring-device@81ad9db90310b4224050b60eaf6fb1feac025119
```

Comment:
Perhaps https://github.com/okta-awscli or https://github.com/Nike-Inc folks can fork this repository.

This way folks wouldn't have to use my fork at all.